src/EditorFeatures/Core/InlineHints/InlineHintsOptionsStorage.cs

```diff
@@ -42,53 +42,53 @@ public static InlineTypeHintsOptions GetInlineTypeHintsOptions(this IGlobalOptio
         //  Parameter hints
 
         public static readonly PerLanguageOption2<bool> EnabledForParameters =
```

Comment:
>static [](http://example.com/codeflow?start=15&length=6)

Would be nice to have one option with flag-enum instead of 8 booleans. Perhaps file an issue and we could decide if it is worth following up on.